lib/tap_monitor.rb

```diff
@@ -45,7 +45,7 @@ def check_dead
     def start_missing
       return unless @running
 
-      BeerTap.all.each do |tap|
+      BeerTap.all.each do |tap| # standard:disable Rails/FindEach
```

Comment:
`find_each` is probably fine, but definitely overkill

Reply:
yeah, same thought.